cmd/kubeadm/app/util/kubeconfig/kubeconfig_test.go

```diff
@@ -345,3 +346,182 @@ func TestHasCredentials(t *testing.T) {
 		})
 	}
 }
+
+func TestGetClusterFromKubeConfig(t *testing.T) {
+	tests := []struct {
+		name      string
+		config    *clientcmdapi.Config
+		expected  string
+		expected1 *clientcmdapi.Cluster
```

Comment:
ok it seems at k/k master it does have the 2 return arguments
https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/util/kubeconfig/kubeconfig.go#L107

please use:
```suggestion
		expectedClusterName  string
		expectedCluster *clientcmdapi.Cluster
```